lib/themes/dosomething/paraneue_dosomething/js/components/ContextSlide.js

```diff
@@ -1,5 +1,9 @@
 const React = require('react');
 
+/**
+ * ContextSlide Component
+ * <ContextSlide />
+ */
 class ContextSlide extends React.Component {
   constructor(props) {
```

Comment:
You can exclude the `constructor` if you're not overriding anything (e.g. setting initial state, etc).